src/platform_impl/linux/window.rs

```diff
@@ -773,15 +779,7 @@ impl Window {
   }
 
   pub fn theme(&self) -> Theme {
-    if let Some(settings) = Settings::default() {
-      let theme_name = settings.gtk_theme_name().map(|s| s.as_str().to_owned());
-      if let Some(theme) = theme_name {
-        if GTK_THEME_SUFFIX_LIST.iter().any(|t| theme.ends_with(t)) {
```

Comment:
If we choose to keep a theme state, we probably don't need this constant.
Can you remove `GTK_THEME_SUFFIX_LIST`'s definition as well?

Reply:
We still need to check using `GTK_THEME_SUFFIX_LIST` because a developer may not specify an explicit theme to be set so it could be either light or dark but this PR implementation assumes that it will always be light theme.